src/components/login-button/index.tsx

```diff
@@ -63,7 +63,6 @@ const LoginButton = () => {
         </Dropdown.Item> */}
         <Dropdown.Item
           key="Near"
-          description="Authenticate with Near Wallet"
```

Comment:
tyvm for removing this unnecessary text from the sign out button @devofficer ; i appreciate your quick response time to this change from today's meeting!